rados/conn.go

```diff
@@ -280,6 +281,38 @@ func (c *Conn) DeletePool(name string) error {
 	}
 }
 
+// TODO
+func (c *Conn) GetPoolByName(name string) (int64, error) {
+	if err := c.ensure_connected(); err != nil {
+		fmt.Println("NOT CONNECTED WHOOPS")
+		return 0, err
+	}
+	c_name := C.CString(name)
+	defer C.free(unsafe.Pointer(c_name))
+	ret := int64(C.rados_pool_lookup(c.cluster, c_name))
+	if ret == -C.ENOENT {
+		return 0, RadosError(ret)
+	} else {
```

Comment:
else is not required as you are returning in if condition

Reply:
Well, this seems to be the coding practice throughout the file.